sonar-project.properties

```
@@ -0,0 +1,33 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_updater
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/updater
+sonar.links.ci=https://drone.owncloud.com/owncloud/updater/
+sonar.links.scm=https://github.com/owncloud/updater
+sonar.links.issue=https://github.com/owncloud/updater/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# Just look in these directories for code
+sonar.sources=.
+sonar.inclusions=appinfo/**,lib/**
```

```suggestion
sonar.inclusions=app/**,pub/**,src/**
```

This repo has code in different folders than the usual oC10 apps